docker-compose.yml

```diff
@@ -0,0 +1,40 @@
+version: '3'
+
+services:
+  app:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    volumes:
+      - .:/app
+    environment:
+      - OPENAI_API_KEY= ${OPENAI_API_KEY:-}
+    ports:
+      - "8000:8000"
+    depends_on:
+      - db
+    restart: always
+    networks:
+      - chatapp-network
```

Comment:
I prefer to use the default network whenever networking doesn't require anything special.